### ISSUE TITLE
Interfaces for extracting instrument events from musical audio

### ISSUE THREAD

**Issue by jtcramer** [marl/urbanorchestra#1]
Create a function or classes that extract events from all stems in the given song

**Comment by lostanlen** [comment]
Do we really need to segment events? MedleyDB already has instrument activations as metadata.
I understand that the VGGish features will be extracted event by event?
In this case we do need an onset-offset detector.

**Comment by jtcramer** [comment]
You're right, though I do remember there being some desire for onset detection on the songs for some reason. But yes, with instrument activations with MedleyDB we can just use those. The default librosa onset detection (which uses spectral flux, though I'm not sure if it's the variant you describe) is currently used for event detection in the urban audio.
